miniwallet/deploy/001_deploy_miniWallet.ts

```diff
@@ -0,0 +1,85 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers } from 'hardhat'
+
+const OPERATOR_ROLE = ethers.utils.id('OPERATOR_ROLE')
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+  const chainId = await getChainId()
+  let initialOperatorThreshold
+  let initialOperators
+  let initialUserLimit
+  let initialAuthLimit
+
+  console.log(`chainId: ${chainId}`)
+  if (chainId === '1666600000,') {
+    console.log('Harmony Mainnet Deploy')
+    initialOperatorThreshold = config.mainnet.miniWallet.initialOperatorThreshold
+    initialOperators = config.mainnet.miniWallet.initialOperators
+    initialUserLimit = config.mainnet.miniWallet.initialUserLimit
+    initialAuthLimit = config.mainnet.miniWallet.initialAuthLimit
+  } else {
+    console.log(`Test Deploy on chainId: ${chainId}`)
+    initialOperatorThreshold = config.test.miniWallet.initialOperatorThreshold
+    initialOperators = config.test.miniWallet.initialOperators
+    initialUserLimit = config.test.miniWallet.initialUserLimit
+    initialAuthLimit = config.test.miniWallet.initialAuthLimit
+  }
+
+  console.log('operators:', JSON.stringify(initialOperators))
+  const deployedContract = await deploy('MiniWallet', {
+    contract: 'MiniWallet',
+    from: deployer,
+    proxy: {
+      owner: deployer,
+      proxyContract: 'EIP173Proxy',
```

Comment:
where is EIP173Proxy?

Reply:
This was a simple proxy used by hardhat-deploy. We have update this now to use `ERC1967Proxy.sol` which is UUPS compatible and is held under the [deployments folder](https://github.com/polymorpher/sms-wallet/tree/miniV0/miniwallet/contracts/deployments)